docs/root/configuration/listeners/network_filters/rbac_filter.rst

```diff
@@ -10,6 +10,10 @@ block-list (DENY) set of policies based on properties of the connection (IPs, po
 This filter also supports policy in both enforcement and shadow modes. Shadow mode won't effect real
 users, it is used to test that a new set of policies work before rolling out to production.
 
+When a request is denied, the :ref:`RESPONSE_DETAILS<config_access_log_format_response_details>`
```

Comment:
How do we know it's the RBAC filter setting this? I.e. don't we both need to know the response details and which network filter sets this?

Reply:
The RBAC filter will set it in the format of `rbac_access_denied_matched_policy[policy_name]`, this should tell which filter sets it. Also updated the doc to make this more clear.